### ISSUE TITLE
"全民瘋AI系列-經典機器學習"頁6-34，程式6.12視覺化kernel='poly'建模結果

### ISSUE THREAD

**Issue by lancelotyang1225** [andy6804tw/2021-13th-ironman#13]
你好

程式6.12最後一行

plt.title(f'C=5\nSVC with polynominal (degree=3) kernel \n Accuracy:{accuracy:.2f}')

可是於程式6.11建模時參數C是設定為1

是否上述程式碼要修正    以符合現況

還是您有想比較C=1及C=5的訓練效果

謝謝

**Comment by andy6804tw** [comment]
@lancelotyang1225 沒錯書上給的程式碼是 C=1，對應的視覺畫圖應該給C=1才對。不卻以優雅的程式設計來說或許可以將這個參數設定成變數`param_c`以免模型跟繪圖的不一致：

```py
from sklearn import svm

# 建立 kernel='poly' 模型
param_c = 1
poly_svc_model=svm.SVC(kernel='poly', degree=3, gamma='auto', C=param_c)
# 使用訓練資料訓練模型
poly_svc_model.fit(X_embedded, y)

# 取得xy座標數據點
X0, X1 = X_embedded[:, 0], X_embedded[:, 1]
# 建立網格點數據
xx, yy = make_meshgrid(X0, X1)
# 繪製決策邊界
plot_contours(plt, poly_svc_model, xx, yy, cmap=plt.cm.Paired, alpha=0.8)
# 繪製散點圖
scatter = plt.scatter(X0, X1, c=y, cmap=plt.cm.Paired, s=20, edgecolors='k')
# 添加圖例
plt.legend(*scatter.legend_elements(), title="Digits", bbox_to_anchor=(1.2, 0.8))
plt.xlabel('T-SNE 1')
plt.ylabel('T-SNE 2')
# 計算準確率
accuracy =poly_svc_model.score(X_embedded, y)
plt.title(f'C={param_c}\nSVC with polynomial (degree 3) kernel\n Accuracy: {accuracy:.2f}')
plt.show()
```

**Comment by lancelotyang1225** [comment]
謝謝大大回應
